init.sh

```diff
@@ -37,6 +37,7 @@ if [ "$install_directory" != "." ]; then
         mkdir -p .github/workflows
     fi
     mv "$install_directory/.github/workflows"/* .github/workflows
+    rmdir -p "$install_directory/.github/workflows"
```

Comment:
Great, thanks! :+1: 
Can you also add `--ignore-fail-on-non-empty` to suppress the warning about `$install_directory` not being empty (and therefore not being removed)?

Reply:
sure thing !! added the flag --ignore-fail-on-non-empty
Thank you